doc/user-guide/template.md

```diff
@@ -64,29 +64,42 @@ The simplest use case is to use a variable to determine which data to filter:
 ### Data filtering
 
 ```{code-cell} ipython3
+from string import Template
+
 sex = "MALE"
+template = Template(
+    """
+SELECT *
+FROM penguins.csv
+WHERE sex = '$sex'
+"""
+)
+
+sex_male = template.substitute(sex=sex)
 ```
 
 ```{code-cell} ipython3
-%%sql
-SELECT *
-FROM penguins.csv
-WHERE  sex = '{{sex}}'
+%sql {{sex_male}}
```

Comment:
I just realized the AC was incorrect and I didn't notice it. we *should not* recommend using `string.Template`, but recommend using JupySQL's native templating capabilities, something like this:


```
sex = "MALE"
````

then, in a different cell:

```
%%sql
SELECT *
FROM penguins.csv
WHERE sex = '{{sex}}'
````

so we have to revert the changes to this file.

and then show something similar to the example I just showed above in this section: https://jupysql.ploomber.io/en/latest/api/magic-sql.html#templated-sql-queries (because such section is using string.Template and we *do not* want that)

and link from here: https://jupysql.ploomber.io/en/latest/api/magic-sql.html#templated-sql-queries

to this file: user-guide/template

Reply:
changed example into using {{variable}} : [fixed](https://github.com/ploomber/jupysql/pull/871/files#diff-c8695088644e6c2bfc26301cbd769831694df7270b90e4057e111e17100ba9cbR295-R303)

added link to `user-guide/template` : [fixed](https://github.com/ploomber/jupysql/pull/871/files#diff-c8695088644e6c2bfc26301cbd769831694df7270b90e4057e111e17100ba9cbR308)